<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_docs</class>
 <widget class="QDialog" name="new_docs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 250, 250);
font-family: Arial</string>
  </property>
  <widget class="QPushButton" name="add_docs">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color:black;
background-color:rgba(220, 220, 220, 30);
border: 1px solid rgb(0, 0, 0);
border-radius: 7px;
}</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>361</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="doc_1_lb">
        <property name="text">
         <string>Документ №1           </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="doc_2_lb">
        <property name="text">
         <string>Документ №2                </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="doc_3_lb">
        <property name="text">
         <string>Документ №3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="doc_4_lb">
        <property name="text">
         <string>Документ №4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="doc_5_lb">
        <property name="text">
         <string>Уровень качества:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="all_type_lb">
        <property name="text">
         <string>Типы сварки:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="doc_1">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.4-083-2006</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.2-649-2012</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.2-136-2007</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ГОСТ 34347-2017</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СП 70.13330.2012</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="doc_2">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.4-083-2006</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.2-649-2012</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.2-136-2007</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ГОСТ 34347-2017</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СП 70.13330.2012</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="doc_3">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.4-083-2006</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.2-649-2012</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.2-136-2007</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ГОСТ 34347-2017</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СП 70.13330.2012</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="doc_4">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.4-083-2006</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.2-649-2012</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СТО Газпром 2-2.2-136-2007</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ГОСТ 34347-2017</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СП 70.13330.2012</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="quality_level">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="all_welding_type">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
